src/components/ctaLink/CTALink.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import styled from 'styled-components'
+import theme from '../../theme/theme'
+import { handleUrl } from '../../utilities'
+
+interface CTALinkProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  /**
+   * URL to navigate to
+   */
+  to: string
+  as?: any
+}
+
+const StyledLink = styled.a<CTALinkProps>`
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/app/PrideInLondon/pride-london-web/pullRequest?prid=4644853)